src/main/java/uk/gov/justice/digital/delius/service/NotificationService.java

```diff
@@ -0,0 +1,72 @@
+package uk.gov.justice.digital.delius.service;
+
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import uk.gov.justice.digital.delius.config.DeliusIntegrationContextConfig;
+import uk.gov.justice.digital.delius.config.DeliusIntegrationContextConfig.IntegrationContext;
+import uk.gov.justice.digital.delius.controller.BadRequestException;
+import uk.gov.justice.digital.delius.data.api.ContextlessNotificationCreateRequest;
+import uk.gov.justice.digital.delius.data.api.NotificationCreateRequest;
+import uk.gov.justice.digital.delius.data.api.NotificationResponse;
+import uk.gov.justice.digital.delius.data.api.deliusapi.NewContact;
+
+import java.util.Optional;
+
+import static java.lang.String.format;
+import static uk.gov.justice.digital.delius.transformers.NotificationCreateRequestTransformer.notificationOf;
+import static uk.gov.justice.digital.delius.utils.DateConverter.toLondonLocalDate;
+import static uk.gov.justice.digital.delius.utils.DateConverter.toLondonLocalTime;
+
+@Service
+@AllArgsConstructor
+public class NotificationService {
+
+    private final ReferralService referralService;
+    private final DeliusApiClient deliusApiClient;
+    private final DeliusIntegrationContextConfig deliusIntegrationContextConfig;
+
+    @Transactional
+    public NotificationResponse notifyContact(final String crn, final Long sentenceId, final NotificationCreateRequest request) {
```

Comment:
Is it worth asserting that the contact type is non-attended (or some better validation)? We do this when creating appointment contacts

Reply:
checking for non attendance type